app/index.js

```diff
@@ -250,31 +253,10 @@ module.exports = yeoman.generators.Base.extend({
 
       this.fs.copyTpl(
         this.templatePath('SecurityConfig.java'),
-        this.destinationPath(srcDir + '/config/SecurityConfig.java'),
+        this.destinationPath(srcDir + '/config/' + this.props.configClassName + '.java'),
```

Comment:
I doubt if we need this custom securityconfig in each service, but leave it for now